src/main/java/edu/byu/cs/autograder/compile/CompileHelper.java

```diff
@@ -92,4 +99,18 @@ private String getMavenError(String output) {
         }
         return builder.toString();
     }
+
+    private void replaceGetMessageTime() throws GradingException {
+        String regex = "return [0-9]*L;";
+        String replace = "return " + WEBSOCKET_MESSAGE_TIME + "L;";
+        Path path = new File(gradingContext.stageRepo(),
+                "/shared/src/test/java/passoffTests/TestFactory.java").toPath();
+        try {
+            String orig = Files.readString(path, StandardCharsets.UTF_8);
+            String replaced = orig.replaceAll(regex, replace);
+            Files.writeString(path, replaced);
+        } catch (Exception e) {
+            throw new GradingException("Failed to modify TestFactory", e);
```

Comment:
Are students allowed to move this file? If not that makes things easier, but either way I think we should give a better error message, being more explicit about what is missing from where

Reply:
Hmm. I wouldn't think so, but I'm not sure. I guess there was a way easier way to change this too.